README.md

```diff
@@ -1,43 +1,56 @@
+a substantial amount of effort is needed to manually encode the circuits, and it is extremely challenging to ensure their correctness.
+As a result, many complex logical computations have not been practically simulated.
 
-Design automation software tools for Topological Quantum Error Correction
+`tqec` provides numerous building blocks based on state-of-the-art protocols, with verified correct circuits implementation for each block.
+These blocks can then be combined to construct large-scale logical computations, enabling the automatic compilation of large-scale computational circuits.
+
+**Note:** This project is under active development and provide no backwards compatibility at current stage.
+
+## Documentation
+
+Documentation is available at <https://qchackers.github.io/tqec/>
 
 ## Installation
 
+Currently, you need to install `tqec` from source:
+
 ```sh
 python -m pip install git+https://github.com/tqec/tqec.git
 ```
 
-For a more in-depth explanation, see the [installation procedure](https://tqec.github.io/tqec/installation.html).
-
-## Quick start
+For a more detailed installation guide and common troubleshooting tips, see the [installation page](https://tqec.github.io/tqec/user_guide/installation.html) in the documentation.
 
-Download the example file [logical_cnot.dae](https://github.com/tqec/tqec/tree/main/docs/media/quick_start/logical_cnot.dae).
+## Basic Usage
 
-You can generate `stim.Circuit` instances representing that computation using
+Here we generate the circuits for a logical CNOT between two logical qubits to demonstrate how to use the tool.
+Refer to [quick start](https://tqec.github.io/tqec/user_guide/quick_start.html) in the documentation for more detailed explanation.
 
 ```py
-from tqec import (
-    BlockGraph, compile_block_graph, annotate_detectors_automatically,
-)
-from tqec.noise_models import NoiseModel
+from tqec import BlockGraph, compile_block_graph
+from tqec.noise_model import NoiseModel
+
+# 1. Construct the logical computation
+block_graph = BlockGraph.from_dae_file("assets/logical_cnot.dae")
 
-block_graph = BlockGraph.from_dae_file("logical_cnot.dae")
+# 2. Get the logical observables of interests and compile the computation
 observables, _ = block_graph.get_abstract_observables()
 compiled_computation = compile_block_graph(block_graph, observables=[observables[1]])
 
+# 3. Generate the `stim.Circuit` of target code distance
 circuit = compiled_computation.generate_stim_circuit(
-    # Can be changed to whatever value you want. Large values will
-    # take a lot of time.
+    # k = (d-1)/2 is the scale factor
+    # Large values will take a lot of time.
     k=2,
     # The noise applied and noise levels can be changed.
     noise_model=NoiseModel.uniform_depolarizing(0.001),
 )
-circuit_with_detectors = annotate_detectors_automatically(circuit)
-print(circuit_with_detectors)
 ```
 
 See the [quick start tutorial](https://tqec.github.io/tqec/quick_start.html) for a
```

Comment:
This is a dead link. should instead be:

https://tqec.github.io/tqec/user_guide/quick_start.html